pkg/cli/tsdump.go

```diff
@@ -51,8 +53,15 @@ Dumps all of the raw timeseries values in a cluster.
 		}
 		defer finish()
 
+		resp, err := serverpb.NewAdminClient(conn).ChartCatalog(ctx, &serverpb.ChartCatalogRequest{})
+		if err != nil {
+			return err
+		}
+
 		tsClient := tspb.NewTimeSeriesClient(conn)
-		stream, err := tsClient.Dump(context.Background(), &tspb.DumpRequest{})
+		stream, err := tsClient.Dump(context.Background(), &tspb.DumpRequest{
+			Names: catalog.AllMetricsNames(resp.Catalog...),
```

Comment:
This should be filled in on the server, so that we use the server's catalog (matters when the cli is old)

Reply:
Nevermind, I was reading this wrong. This is already pulling them from the server so all good